flask_prom/__init__.py

```diff
@@ -2,7 +2,7 @@
 from flask import request
 from prometheus_client import Counter, Histogram
 from prometheus_client import start_http_server, make_wsgi_app
-from werkzeug.wsgi import DispatcherMiddleware
+from werkzeug.wsgi.middleware.dispatcher import DispatcherMiddleware
```

Comment:
Should be `from werkzeug.middleware.dispatcher import DispatcherMiddleware`. 0.16.1 fixed the typo in the deprecation message.